ocfweb/docs/templates/docs/servers.html

```diff
@@ -3,6 +3,7 @@
 {% block content %}
     <div class="ocf-content-block">
       <p>At the OCF, there are four physical servers whose job is to host virtual servers. These physical servers are hypervisors running QEMU and KVM. The servers which people see on a daily basis, like the login server, the web server, the print server, and things like that are all virtual machines hosted on these hypervisors.</p>
+      <p>The OCF has multiple physical machines. Some of these are hypervisors, which host virtual servers running on QEMU and KVM. We also have servers that are workers for our Kubernetes cluster, and others comprise our High Performance Computing cluster.
```

Comment:
> The OCF has multiple physical machines.

Nit: could probably be copyedited to come earlier

Reply:
ahh i messed up and forgot to remove the preceding paragraph lol